Packs/Office365AndAzureAuditLog/Integrations/MicrosoftPolicyAndComplianceAuditLog/README.md

```diff
@@ -21,6 +21,13 @@ Use the integration to get logs from the O365 service.
     3. Run the ***o365-auditlog-auth-test*** command to verify that the authorization process was implemented correctly.
 
 
+## Required Permissions To Search Audit Logs
+- The minimum required exchange permissions are **Audit Logs** or **View-Only Audit Logs**.
```

Comment:
Isn't "Exchange" a reference to the Microsoft product? If so, it should be capitalized.

Reply:
Exchange here refer to a section where you can give users in your organization roles/permissions to access resources. it is a section in the Microsoft 365 admin center. 

![image](https://user-images.githubusercontent.com/53861351/153832478-4401aa03-0f62-4500-b05d-308bfbac46d1.png)

Does it still need to be with capital letter?

Reply:
It refers to the product - so yes - please change it.

Reply:
done